DOCKER.md

```diff
@@ -0,0 +1,59 @@
+## Installation on Docker
+
+### Clone the repository
+
+```
+git clone https://github.com/martomi/chiadog.git
+cd chiadog
+```
+
+### Build the Docker image
+
+#### Specific version
+
+```
+docker image build -t chiadog:latest .
+```
+
+#### Latest version
+
+```
+docker image build -t chiadog:0.5.0 --build-arg BRANCH=v0.5.0 .
```

Comment:
Is there a way we can have this such that updates with every new version of chiadog are not necessary? E.g. using main branch and `latest` tag 🤔 Perhaps fetch version from a VERSION file in the root of the repository that we can maintain at central place?

Reply:
Build instructions for a specific version and the latest version were inverted. By default, if BRANCH argument is not supplied, the main branch is used so the image will use the latest version at the time of the build.